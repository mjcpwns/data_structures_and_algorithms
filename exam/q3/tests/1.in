nV: 9
src: 0
srcViralLoad: 70
0 2
0 3
1 3
1 4
1 5
2 3
2 5
3 4
4 5
5 8
7 8

