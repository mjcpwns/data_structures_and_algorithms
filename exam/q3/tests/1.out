nV: 9
src: 0
srcViralLoad: 70

Edge inserted: 0-2
Edge inserted: 0-3
Edge inserted: 1-3
Edge inserted: 1-4
Edge inserted: 1-5
Edge inserted: 2-3
Edge inserted: 2-5
Edge inserted: 3-4
Edge inserted: 4-5
Edge inserted: 5-8
Edge inserted: 7-8

Viral load:
 trasmissionArray[0] is     70.000 
 trasmissionArray[1] is     25.200 
 trasmissionArray[2] is     42.000 
 trasmissionArray[3] is     42.000 
 trasmissionArray[4] is     25.200 
 trasmissionArray[5] is     25.200 
 trasmissionArray[6] is     -1.000 
 trasmissionArray[7] is      0.000 
 trasmissionArray[8] is     15.120 
